src/app/content/components/popUp/ChapterFilter.tsx

```diff
@@ -51,62 +54,198 @@ const chunk = <T extends any>(sections: T[]) => {
   return [sections.slice(0, cutoff), sections.slice(cutoff)].filter((arr) => arr.length > 0);
 };
 
-interface Props {
+export type LocationFilters = Map<
+  string,
+  { section: LinkedArchiveTree | LinkedArchiveTreeSection, children?: LinkedArchiveTreeSection[] }
+>;
+
+interface ChapterFilterProps {
   className?: string;
   disabled?: boolean;
-  locationFilters: HighlightLocationFilters;
+  locationFilters: LocationFilters;
   locationFiltersWithContent: Set<string>;
   selectedLocationFilters: Set<string>;
-  setFilters: (filters: Pick<SummaryFilters, 'locationIds'>) => void;
+  multiselect: boolean;
+  setFilters: (filters: { locationIds: string[] }) => void;
 }
 
 // tslint:disable-next-line:variable-name
-const ChapterFilter = ({
-  className,
-  disabled,
-  locationFilters,
-  locationFiltersWithContent,
-  selectedLocationFilters,
-  setFilters,
-}: Props) => {
+const ChapterFilter = (props: ChapterFilterProps) => {
+  const [isOpenChapterId, setIsOpenChapterId] = React.useState<string | null>(null);
```

Comment:
should `isOpenChapterId` be `openChapterId`, its not like a boolean or something right?